src/Trakx.Circle.ApiClient/nswag.json

```diff
@@ -55,7 +56,7 @@
       "wrapResponses": true,
       "wrapResponseMethods": [],
       "generateResponseClasses": true,
-      "responseClass": "Response",
```

Comment:
here we should use the one from [Trakx.Utils.Apis](https://github.com/trakx/dotnet-utils/blob/dev/src/Trakx.Utils/Apis/Response.cs)